StringThings/Extensions/IgnoreCaseStringExtensions.cs

```diff
@@ -12,7 +12,7 @@ private static void ThrowIfArgumentNull(string argName, string? s)
 
         public static bool EqualsIgnoreCase(this string? s, string? other)
         {
-            ThrowIfArgumentNull(nameof(s), s);
+            ThrowIfArgumentNull(nameof(s), s);            

```

Comment:
I took out this whitespace and the tests still pass 😉

Reply:
Just practicing social distancing for COVID.